src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Windows.cs

```diff
@@ -14,8 +15,8 @@ internal static class NameResolutionPal
         // used by GetHostName() to preallocate a buffer for the call to gethostname.
         //
         private const int HostNameBufferLength = 256;
-        private static volatile bool s_Initialized;
-        private static object s_InitializedLock = new object();
+        private static bool s_initialized;
```

Comment:
@pgavlin If the tool already lower-cases the first letter after `s_`, then we should update the docs to add an example. I don't have a preference for casing but I like consistency. Took a note to track this.